lighthouse-core/gather/gather-runner.js

```diff
@@ -631,8 +640,7 @@ class GatherRunner {
     // Go to about:blank, set up, and run `beforePass()` on gatherers.
     await GatherRunner.loadBlank(driver, passConfig.blankPage);
     await GatherRunner.setupPassNetwork(passContext);
-    const isPerfPass = GatherRunner.isPerfPass(passContext);
-    if (isPerfPass) await driver.cleanBrowserCaches(); // Clear disk & memory cache if it's a perf run
```

Comment:
Also, this comment was not useful.

Reply:
it is indeed a [relic of a time](https://github.com/GoogleChrome/lighthouse/commit/740c2e99d4f48118004da6ce2f2fa3308a95c911#diff-42278a7ee772120215ea8cfe1b0cb1b1R213) when gather-runner was inscrutable and lines needed to be explained, but a reminder that it's intended to be disk *and* memory cache seems somewhat useful to me :)

honestly though I'm not sure why this isn't in `setupPassNetwork` since it's exactly supposed to set the network state

Reply:
`Clear disk & memory cache if it's a perf run` says three things:

1. clear disk cache
2. clear memory cache
3. do these things only if perf run

all of which is readily apparent, either via the short jsdocs or the names of these methods.

this is getting nitty so I just put it back. Just doing the rename.

Reply:
fwiw I wasn't trying to argue that it must stay, just wanted to support why it existed and that it's not 100% devoid of value :)

but it's w/e